### DISCUSSION TITLE
Allow users to select which daily notes plugin to use when both are active

### DISCUSSION THREAD

**Discussion by MMoMM-org** [davish/obsidian-full-calendar#372] [Feature Requests]
Daily Notes (for Dayplanning / Scheduling) and Periodic Notes (for Journaling) at the same time.

If I enable the Daily Notes option it will pick the Periodic Notes.

I would like to have all my events etc in my Daily Notes though.. as those notes also contain my schedule.

I would suggest adding a dropdown to select the plugin to use (there might come more in the future).

**Comment by davish** [comment]
Thanks for the feature suggestion! I don't know how difficult this would be, considering I'm leveraging https://github.com/liamcain/obsidian-daily-notes-interface for the daily notes interactions.

**Comment by deurk** [comment]
Would be a wonderful addition as a lot of people disable the core `Daily Note` to use `Periodic Notes`.

In my use case for example, I have an external service that creates a Full Calendar event note per time entries in a time log application, which adds up to 20, 30 or sometimes more per day. Not being able to have the daily note usage with Periodic Notes means I have to have as many files when it could be just one per day with it. It then complicates synchronizing things (which takes much longer given there are 30x more files to handle) etc.

Can't wait for it, to be honest!

(Oh, and happy new year and thank you for the great work thus far!)
